### ISSUE TITLE
After last git pull Cannot find streamer site

### ISSUE THREAD

**Issue by JoshWho** [WWBN/AVideo#4945]
![9f861b87-5a72-497c-aea0-965c976aa6ac](https://user-images.githubusercontent.com/39392449/114797837-a58c0800-9d59-11eb-9d3d-931daca59916.png)

**Comment by DanielnetoDotCom** [comment]
check your encoder logs

probably it cannot connect to your streamer

**Comment by JoshWho** [comment]
what changed?

**Comment by JoshWho** [comment]
Im live streaming at the moment why would that have a change in it?

**Comment by JoshWho** [comment]
What directory am I looking for?

**Comment by JoshWho** [comment]
I see this in apache when trying to log in

`2601:248:8301:a230:edec:8225:8e60:ca25 - - [14/Apr/2021:23:16:23 -0500] "POST /upl6/login HTTP/1.1" 200 4380 "https://ww2.joshwhotv.com/upl6/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.114 Safari/537.36"
`

**Comment by JoshWho** [comment]
```
root@mail3:~# tail -f /var/log/apache2/error.log
[Wed Apr 14 23:03:28.380087 2021] [core:crit] [pid 20252] (13)Permission denied: [client 100.8.189.114:31924] AH00529: /var/www/html/YouPHPTube/videos/.htaccess pcfg_openfile: unable to check htaccess file, ensure it is readable and that '/var/www/html/YouPHPTube/videos/' is executable, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:03:28.447118 2021] [core:crit] [pid 20253] (13)Permission denied: [client 100.8.189.114:25418] AH00529: /var/www/html/YouPHPTube/videos/.htaccess pcfg_openfile: unable to check htaccess file, ensure it is readable and that '/var/www/html/YouPHPTube/videos/' is executable, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:03:47.069639 2021] [php7:error] [pid 20402] [client 100.8.189.114:41002] script '/var/www/html/YouPHPTube/view/theme.css.php' not found or unable to stat, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:03:47.253699 2021] [core:crit] [pid 20250] (13)Permission denied: [client 100.8.189.114:30170] AH00529: /var/www/html/YouPHPTube/videos/.htaccess pcfg_openfile: unable to check htaccess file, ensure it is readable and that '/var/www/html/YouPHPTube/videos/' is executable, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:03:47.313425 2021] [core:crit] [pid 20341] (13)Permission denied: [client 100.8.189.114:41290] AH00529: /var/www/html/YouPHPTube/videos/.htaccess pcfg_openfile: unable to check htaccess file, ensure it is readable and that '/var/www/html/YouPHPTube/videos/' is executable, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:12:05.981358 2021] [php7:error] [pid 20402] [client 185.69.144.20:28256] script '/var/www/html/YouPHPTube/view/theme.css.php' not found or unable to stat, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:12:18.793867 2021] [php7:error] [pid 20254] [client 185.69.144.20:23410] script '/var/www/html/YouPHPTube/view/theme.css.php' not found or unable to stat, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:12:19.253966 2021] [php7:error] [pid 20402] [client 185.69.144.20:62466] script '/var/www/html/YouPHPTube/view/theme.css.php' not found or unable to stat, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:12:59.184761 2021] [php7:error] [pid 20253] [client 185.69.144.20:63976] script '/var/www/html/YouPHPTube/view/theme.css.php' not found or unable to stat, referer: https://ww2.joshwhotv.com/upl3/
[Wed Apr 14 23:13:52.419238 2021] [php7:error] [pid 20250] [client 185.69.144.20:11856] script '/var/www/html/YouPHPTube/view/theme.css.php' not found or unable to stat, referer: https://ww2.joshwhotv.com/upl3/


```

**Comment by JoshWho** [comment]
I normally have to use this line in config and I think it is getting ignored

`$global['forceStreamerSiteURL'] = 'https://www.joshwhotv.com/';`

**Comment by milehighkaraoke** [comment]
Try it without the www 
![Screenshot_20210415-004035_Firefox](https://user-images.githubusercontent.com/70327833/114825278-3c20ef00-9d83-11eb-884f-75da353e9dc5.jpg)

**Comment by milehighkaraoke** [comment]
If that don't work enter it with http to your local address  tube.yoursite.con

**Comment by DanielnetoDotCom** [comment]
I went to your encoder server and I have `wget `to your login page

```
# wget -O test.json 'https://www.joshwhotv.com/login?user=***&pass=****&encodedPass=true'
--2021-04-15 07:31:09--  https://www.joshwhotv.com/login?user=****&pass=***&encodedPass=true
Resolving www.joshwhotv.com (www.joshwhotv.com)... 104.31.87.238
Connecting to www.joshwhotv.com (www.joshwhotv.com)|*04.*1.*7.2*8|:443... failed: No route to host.
```

I have checked your streamer server and the IP is different than the resolved one 1*4.2*7.1*0.1*7

I checked your hosts file and you had some entries to the wrong IP, I have added the correct IP and now it is working fine

**Comment by JoshWho** [comment]
very interesting. WIll run some security check ups now. Seems a little odd that happened but I am glad it is working again. Thank you.
